trin-core/src/portalnet/types/content_key.rs

```diff
@@ -658,4 +715,73 @@ mod test {
         assert_eq!(hex::decode(expected_content_key).unwrap(), encoded);
         assert_eq!(expected_content_id, key.content_id());
     }
+
+    // test values sourced from: https://github.com/ethereum/portal-network-specs/blob/master/content-keys-test-vectors.md
+    #[test]
+    fn epoch_accumulator_key() {
+        let epoch_hash =
+            hex::decode("e242814b90ed3950e13aac7e56ce116540c71b41d1516605aada26c6c07cc491")
+                .unwrap();
+        let expected_content_key =
+            hex::decode("03e242814b90ed3950e13aac7e56ce116540c71b41d1516605aada26c6c07cc491")
+                .unwrap();
+        let expected_content_id =
+            &hex::decode("9fb2175e76c6989e0fdac3ee10c40d2a81eb176af32e1c16193e3904fe56896e")
+                .unwrap();
+
+        let content_key = HistoryContentKey::EpochAccumulator(EpochAccumulator {
+            epoch_hash: H256::from_slice(&epoch_hash),
+        });
+        assert_eq!(&content_key.content_id().to_vec(), expected_content_id);
+
+        let actual_content_key: Vec<u8> = content_key.clone().into();
+        assert_eq!(actual_content_key, expected_content_key);
```

Comment:
I found `content_key` vs `actual_content_key` didn't help me understand what was happening. This would have helped me more:
```suggestion
        let encoded_content_key: Vec<u8> = content_key.clone().into();
        assert_eq!(encoded_content_key, expected_content_key_encoding);
```

(Same in the other 2 tests)